Evaluation_1/ShuffleNet/export.py

```diff
@@ -0,0 +1,48 @@
+import onnx
+from onnx_tf.backend import prepare
+import tensorflow as tf
+import tensorflow_hub as hub
+import os
+import io
+import requests
+from tqdm import tqdm
+import tarfile
+
+DOWNLOAD_LINK = "https://s3.amazonaws.com/download.onnx/models/opset_8/shufflenet.tar.gz"
+
+
+if not os.path.exists(DOWNLOAD_LINK.split("/")[-1]):
+  response = requests.get(DOWNLOAD_LINK, stream=True)
+  with open(DOWNLOAD_LINK.split("/")[-1], "wb") as handle:
+    for data in tqdm(response.iter_content(chunk_size=io.DEFAULT_BUFFER_SIZE)):
+      handle.write(data)
+  tar = tarfile.open(DOWNLOAD_LINK.split("/")[-1])
+  tar.extractall()
+  tar.close()
+
+
+if not os.path.exists("shufflenet.pb"):
+  model = onnx.load("shufflenet/model.onnx")
+  tf_rep = prepare(model)
+  tf_rep.export_graph("shufflenet.pb")
+
+
```

Comment:
For readability, try to restructure files as:
1] All function definitions & helpers
2] An `if __name__ == "__main__"` block (that defines what the binary does).

Also, try to have a max of 2 newlines between two functions/blocks.